modules/tinymce/src/core/main/ts/undo/Levels.ts

```diff
@@ -5,23 +5,17 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { Arr, Cell, Optional } from '@ephox/katamari';
+import { Arr, Thunk } from '@ephox/katamari';
 import { Html, Remove, SelectorFilter, SugarElement } from '@ephox/sugar';
 
 import Editor from '../api/Editor';
 import * as TrimHtml from '../dom/TrimHtml';
 import * as Fragments from './Fragments';
 import { UndoLevel, UndoLevelType } from './UndoManagerTypes';
 
-const undoLevelDocument = Cell<Optional<Document>>(Optional.none());
```

Comment:
This only needed a `Thunk.cached` and didn't need a cell/singleton so I've replaced it.

Reply:
oh wow I guess when we converted to katamari the person doing this file didn't know about cached thunks :)